docs/docs/api/cpp/cppstart.md

```diff
@@ -0,0 +1,8 @@
+---
+slug: /cppstart
```

Comment:
This changes the route of this page to `root/cppstart` instead of the root/api/cpp/cppstart`. I'm ok changing the route itself but we should keep it under the /api base